src/MeshData.cpp

```diff
@@ -43,25 +43,25 @@ void MeshData::clear()
     this->vertices.clear();
 }
 
-PyObject* MeshData::getVerticesAsBytes()
+std::vector<char> MeshData::getVerticesAsBytes()
 {
-    std::string vertices_data;
+    std::vector<char> vertices_data;
```

Comment:
Best also add a `reserve(vertices.size() * sizeof(float))` here, since we know precisely what size of vector we need.